pillar.example

```diff
@@ -18,6 +18,9 @@ hana:
   # hana_extract_dir should be a new directory and seperate from location where the compressed files are present, to avoid conflicts in file permissions.
   hana_archive_file: '/sapmedia/51053492.ZIP'
   hana_extract_dir: '/sapmedia_extract/HANA'
+  # The HANA Client sar archive is needed for monitoring if HANA DB sar archive is provided. It will be extracted to hana_client_extract_dir path (optional, by default /sapmedia_extract/HANA_CLIENT)
+  client_archive_file: '/sapmedia/IMDB_CLIENT20_003_144-80002090.SAR'
```

Comment:
Maybe they should be `hana_client_archive_file` and `hana_client_extract_dir`?
In fact, the comment above uses that syntax.
To make it more consistent